neo4j-jdbc-translator/impl/src/main/java/org/neo4j/jdbc/translator/impl/SqlToCypherConfig.java

```diff
@@ -48,23 +48,29 @@ public final class SqlToCypherConfig {
 
 	/**
 	 * A constant property name to make the translator always escape literal names.
+	 * @deprecated no replacement
```

Comment:
Don't see the reference to this PR. Maybe another commit to have it more visible or is this a sneaky deprecation? (And I shouldn't have commented on this to leave it as stealth as possible, sorry)